packages/server-side-render/src/server-side-render.js

```diff
@@ -221,7 +227,7 @@ export default function ServerSideRender( props ) {
 
 	if ( isLoading ) {
 		return (
-			<LoadingResponsePlaceholder { ...props } showLoader={ showLoader }>
+			<LoadingResponsePlaceholder { ...props } isLoading={ isLoading }>
```

Comment:
Minor, but if we're worried about back-compat, I can revert the prop name, or just add a changelog entry about it.

Reply:
I think [this component is rendered when `isLoading` is `true`](https://github.com/WordPress/gutenberg/pull/70147/files?diff=unified&w=0#diff-1fcc38b89edc0d5f7b6239481f42d74b8e2010d792fb26b1e0f8623b222a06fbR228), so won't the value of the `isLoading` prop always be `true`?

Reply:
That's true, mostly passing the value as a safeguard for future changes. Same as here - https://github.com/WordPress/gutenberg/pull/70147#discussion_r2092850652.